inference/core/workflows/core_steps/loader.py

```diff
@@ -343,6 +368,46 @@
     "allowed_write_directory": WORKFLOW_BLOCKS_WRITE_DIRECTORY,
 }
 
+KINDS_SERIALIZERS = {
+    IMAGE_KIND.name: serialise_image,
```

Comment:
```suggestion
    IMAGE_KIND.name: serialize_image,
```

Also all other lines containing `serialise`